loadgen/scenario.go

```diff
@@ -149,6 +146,10 @@ const DefaultMaxConcurrent = 10
 type RunConfiguration struct {
 	// Number of iterations to run of this scenario (mutually exclusive with Duration).
 	Iterations int
+	// StartFromIteration is the iteration to start from when resuming a run.
+	// This is used to skip iterations that have already been run.
+	// Default is zero. If Iterations is set, too, must be less than or equal to Iterations.
+	StartFromIteration int
```

Comment:
Moved StartFromIteration here since it seem it's really a configuration setting. (isn't it?)